### PR TITLE
s()  called inside mgcv::gam

### PR DESCRIPTION
**Pull request by Simontuk**
I tried to find a solution for a small namespace problem within the function mgcv::gam() in the Variance adjustment. 
If the function s() is defined by another package or by the user in the global environment, adjustVariance will not work.
As it is not possible to call mgcv::gam(v~mgcv::s(m, k=gam.k))  I suggest this solution.

### CODE REVIEW

File: R/sparseclust.r [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #' @import igraph
 #' @importFrom irlba irlba
 #' @import pcaMethods
-#' @importFrom mgcv gam
+#' @importFrom mgcv gam s
 #' @importFrom parallel mclapply
 # @importFrom scde pagoda.reduce.loading.redundancy pagoda.reduce.redundancy
```
Comment (slowkow) [inline R/sparseclust.r]: You might want to change `#` to `#'` on line 13.

**Comment by barkasn** [comment]
haven't been following this alot, do you have a definitive solution to commit to the main branch?

File: R/sparseclust.r (hunk 1 above)
Comment (barkasn) [inline R/sparseclust.r]: Actually I turned this into a comment when removed the scde dependency. The functions are now available in p2 directly.

Comment (slowkow) [inline R/sparseclust.r]: Thanks for the clarification! I thought it was a typo.

**Pull request merged**